test/test-v8debugapi.ts

```diff
@@ -670,10 +696,12 @@ describe('v8debugapi', function() {
             return m.name === 'versions' && m.varTableIndex;
           } as any));
 
-          api.clear(bp);
           config.capture.maxDataSize = oldMaxData;
```

Comment:
The same concern about moving these lines inside the callback of `api.clear` as described above.